pkg/framework/resource/resource.go

```diff
@@ -96,6 +96,23 @@ func (r *Resource) InstallResource(name, values, channel string, conditions ...f
 	return nil
 }
 
+func (r *Resource) UpdateResource(name, values, channel string, conditions ...func() error) error {
+	chartValuesEnv := os.ExpandEnv(values)
+	chartname := fmt.Sprintf("%s-chart", name)
+
+	tarballPath, err := r.apprClient.PullChartTarball(chartname, channel)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	err = r.helmClient.UpdateReleaseFromTarball(name, tarballPath, helm.UpdateValueOverrides([]byte(chartValuesEnv)))
```

Comment:
What happens if `chartValuesEnv` is empty? Will this still work?

Reply:
Yes, IIRC we actually have to pass _at least_ an empty string in here. If the string is empty, then the defaults inside the chart will be used.